solarforecastarbiter/metrics/tests/test_deterministic.py

```diff
@@ -82,3 +82,49 @@ def test_r2(y_true, y_pred, value):
 def test_crmse(y_true, y_pred, value):
     crmse = deterministic.centered_root_mean_square(y_true, y_pred)
     assert crmse == value
+
+
+@pytest.mark.parametrize("y_true,y_pred,value", [
+    ([0, 1], [0, 1], 0.0),
+    ([1, 2], [1, 2], 0.0),
+    ([0, 1], [0, 2], 0.5),
+    ([0, 1, 2], [0, 0, 2], 1.0 / 3.0),
+])
+def test_ksi(y_true, y_pred, value):
+    ksi = deterministic.kolmogorov_smirnov_integral(y_true, y_pred)
+    assert pytest.approx(ksi) == value
+
+
+@pytest.mark.parametrize("y_true,y_pred,value", [
+    ([0, 1], [0, 1], 0.0),
+    ([1, 2], [1, 2], 0.0),
+])
+def test_ksi_norm(y_true, y_pred, value):
+    ksi = deterministic.kolmogorov_smirnov_integral(
+        y_true, y_pred, normed=True
+    )
+    assert ksi == value
+
+
+@pytest.mark.parametrize("y_true,y_pred,value", [
+    ([0, 1], [0, 1], 0.0),
+    ([1, 2], [1, 2], 0.0),
+    ([0, 1, 2, 3, 4], [0, 0, 0, 0, 0], 0.8 - 1.63 / np.sqrt(5)),
```

Comment:
agree this is the correct value.

Reply:
Thanks for taking the time to check, Cliff.